sigh/lib/assets/resign.sh

```diff
@@ -639,6 +639,71 @@ function resign {
         # Start with using what comes in provisioning profile entitlements before patching
         cp -f "$PROFILE_ENTITLEMENTS" "$PATCHED_ENTITLEMENTS"
 
+        log "Removing blacklisted keys from patched profile"
+        # See https://github.com/facebook/buck/issues/798 and https://github.com/facebook/buck/pull/802/files
+
+        # Update in https://github.com/facebook/buck/commit/99c0fbc3ab5ecf04d186913374f660683deccdef
+        # Update in https://github.com/facebook/buck/commit/36db188da9f6acbb9df419dc1904315ab00c4e19
+
+        # Buck changes referenced above are not self-explanatory and do not seem exhaustive or up-to-date
+        # Comments below explain the rules applied to each key in order to make alignment with future Xcode export logic easier
+        BLACKLISTED_KEYS=(\
+            "com.apple.developer.icloud-container-development-container-identifiers" \
+            # PP list identifiers inconsistent with app-defined ones and this key does not seem to appear in IPA entitlements, so ignore it
```

Comment:
I don't want to sound picky, but only about an hour after reading this code I realized that the description for each key goes _below_ the key :)

Got me confused, since I got used to code comments being put _before_ the code, so all this time I thought `com.apple.developer.icloud-container-environment` does not appear in IPA entitlements, while it does.

Would you mind changing the code/comment order here?